### DISCUSSION TITLE
Adding filters for an expectation

### DISCUSSION THREAD

**Discussion by n-karahan-afacan** [great-expectations/great_expectations#6507] [Ideas]
I could not locate such a feature in docs. I would like to be able to add a filter for an expectation, ie values should be non-null when column A is <value>.

**Comment by talagluck** [comment]
Hi @n-karahan-afacan - you are describing [Conditional Expectations.](https://docs.greatexpectations.io/docs/reference/expectations/conditional_expectations/)
